api/cpp/include/slint-platform.h

```diff
@@ -148,8 +146,8 @@ class WindowAdapter
     ///
     /// Re-implement this function to forward the call to show/hide the native window
     ///
-    /// When the window becomes visible, this is a good place to send
-    /// slint::Window::dispatch_resize_event and slint::Window::dispatch_scale_factor_change_event
+    /// When the window becomes visible, this is a good place to send the first
+    /// slint::Window::dispatch_scale_factor_change_event
```

Comment:
"When" means "time, "Where" means "place" :)

That said, I wonder why we wait until `set_visible` - why don't we recommend doing this earlier?

AFAICS either the scale factor is "constant" (only one screen) and known by the time we establish a connection to the windowing system (time we create the native window, which means the adapter constructor).
The window manager - at creation time - might already have "assigned" a screen to it and report the correct value. But if not, then only after the window is mapped, the scale factor will change from the "system default" (primary monitor?) and then a separate event is dispatched.

```suggestion
    /// When the window becomes visible, this is a good time to call
    /// slint::Window::dispatch_scale_factor_change_event to initialise the scale factor.
```